lib/chitter_homepage.rb

```diff
@@ -0,0 +1,26 @@
+require 'pg'
+
+class ChitterPage
+  def self.view
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    total = connection.exec("SELECT * FROM peeps ORDER BY id DESC;")
```

Comment:
You can also use the `SELECT * FROM peeps ORDER BY date_posted DESC;` command here to call the peeps by descending order directly from the database.